app/services/rebate_forms_service.rb

```diff
@@ -5,6 +5,28 @@ def initialize(rebate_form_attributes)
     @rebate_form_attributes = rebate_form_attributes
   end
 
+  def generate_qr(rebate_form)
```

Comment:
I still think QR code generation should be in its own service, as it's not part of creating or updating the RebateForm ...

Reply:
But I will 🐘 this for the time being for expedience's sake ... 😄

Reply:
:pray: thanks you